### PR TITLE
Compatibility with browserify

### PR DESCRIPTION
**Pull request by bneigher**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+function compile (watch) {
+  var bundler = browserify({
+    // Required watchify args
+    cache: {}, packageCache: {},
+    // Specify the entry point of your app
+    entries: ['./lib/ptolemy.js'],
+    // Add file extentions to make optional in your requires
+    extensions: ['.js'],
+    // Enable source maps!
+    debug: true,
```
Comment (dremonkey) [inline gulpfile.js:20]: is this needed?

**Review by dremonkey — COMMENTED** [review]
LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### v1.2.0
```
Comment (rbdr) [inline CHANGELOG.md]: This should not be a version in the feature branch (It is a version until it is in the release branch)

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+var gulp = require('gulp');
```
Comment (rbdr) [inline gulpfile.js]: We should favor const and let, and we should capitalize requires

File: lib/ptolemy.js [hunk 4]
```diff
@@ -1,9 +1,13 @@
 'use strict';
 
+
 const Errors = require('./errors');
 const UnknownCRSError = Errors.UnknownCRSError;
 
 const Logger = console;
+
+const epsgio = require('./adapters/epsgio');
```
Comment (rbdr) [inline lib/ptolemy.js]: Capitalize requires

File: package.json [hunk 5]
```diff
@@ -27,12 +27,23 @@
...
+    "derequire": "^2.0.3",
     "eslint-bootstrap": "github:skycatch/eslint-bootstrap",
-    "lab": "^11.1.0"
+    "gulp": "^3.9.1",
+    "gulp-derequire": "^2.1.0",
+    "gulp-notify": "^2.2.0",
+    "gulp-rename": "^1.2.2",
+    "lab": "^11.1.0",
+    "vinyl-source-stream": "^1.1.0",
+    "watchify": "^3.7.0"
```
Comment (rbdr) [inline package.json:47]: Are all of these new devDependencies needed? D:

**Review by rbdr — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (bneigher) [reply package.json:47]: for building the lib yea :/ but assuming when this is installed on a product it's being installed with npm install --production it's not really an issue

File: gulpfile.js (hunk 1 above)
Comment (bneigher) [reply gulpfile.js:20]: removing

File: package.json (hunk 5 above)
Comment (rbdr) [reply package.json:47]: cool

File: gulpfile.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const gulp = require('gulp');
```
Comment (rbdr) [inline gulpfile.js]: Still need caps on the requires

**Review by rbdr — CHANGES_REQUESTED** [review]
Just the capitalization of requires

**Pull request merged**